code/src/java/pcgen/io/Compatibility.java

```diff
@@ -132,8 +142,7 @@ else if (!max.equals(min))
 			return null;
 		}
 		context.commit();
-		PersistentTransitionChoice<?> ptc = applied.getListFor(ListKey.ADD).get(0);
-		return ptc;
+		return applied.getListFor(ListKey.ADD).get(0);
```

Comment:
I'm concerned this won't compile.  There have been similar changes made in other places that when inlined cause various compilers to complain... At what level has this been tested?